src/components/Beers.js

```diff
@@ -0,0 +1,51 @@
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import axios from "axios";
+import BeerDetails from "./BeerDetails";
+
+function Beers() {
+  const [beers, setBeers] = useState([]);
+  const [fetching, setFetching] = useState(true);
+
+  useEffect(() => {
+    axios.get("https://ih-beers-api2.herokuapp.com/beers").then((response) => {
+      setBeers(response.data);
+      setFetching(false);
+    });
+  }, []);
+
+  return (
+    <>
+      <header>
+        <Link to="/">
+          <i>home</i>
+        </Link>
+      </header>
```

Comment:
You have created Header component - so should be rendering it here, instead of writing same code again ;)